components/Layout.js

```diff
@@ -123,7 +123,7 @@ const Layout = ({
         </Head>
         <Header height={theme.headerHeight} {...headerData} />
         <Content>
-          {children}
+          {React.cloneElement(children, { locale })}
```

Comment:
So by doing this, you pass to all of the layout children `locale`?

Reply:
Yes. The pages can now do locale-specific stuff with it.